pages/playground/index.mdx

```diff
@@ -0,0 +1,10 @@
+<Button>Click Me</Button>
+
+
+<ContainerLayout>
+
```

Comment:
A line-space is ***necessary*** if you want to render MDX as the children of a Component